### ISSUE TITLE
Defunct?

### ISSUE THREAD

**Issue by bpj** [pandoc-extras/pandoc-nightly#7]
There has been no release in 9 days and some of the latest releases are with a 2 day interval. What's happening?

**Comment by ickc** [comment]
I think this project is effectively useless now that pandoc uses GitHub Action. Through GitHub Action you can directly pull nightly build binaries from there. See if @jgm agrees.

However I just fixed the macOS build and remove the Windows build and add an arm64 build. Linux and macOS should build fine now but am still trying to get arm64 build working. I guess the Travis CI would still function as a 3rd party fall back nightly build service until it dies.

**Comment by jgm** [comment]
I agree -- we have nightlies now in the main pandoc repo through actions, so this could be wound down.

**Comment by ickc** [comment]
By the way, may be we should make it more apparent for contributors. @jgm, do you want me to expand <https://github.com/jgm/pandoc/blob/master/CONTRIBUTING.md> pointing out nightly is available?

Edit: I mean official nightly builds are available and perhaps pointing out how to retrieve them.

**Comment by jgm** [comment]
Good idea!
